phoenix-core-client/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -1776,7 +1778,18 @@ public MutationState createIndex(CreateIndexStatement statement, byte[][] splits
             return buildIndexAtTimeStamp(table, statement.getTable());
         }
 
-        return buildIndex(table, tableRef);
+        MutationState state = buildIndex(table, tableRef);
+        // If client is validating LAST_DDL_TIMESTAMPS, parent's last_ddl_timestamp changed
+        // so remove it from client's cache. It will be refreshed when table is accessed next time.
+        boolean lastDDLTimestampValidationEnabled =
```

Comment:
Do you want to use [ValidateLastDDLTimestampUtil#getValidateLastDdlTimestampEnabled](https://github.com/apache/phoenix/blob/PHOENIX-6883-feature/phoenix-core-client/src/main/java/org/apache/phoenix/util/ValidateLastDDLTimestampUtil.java#L73C27-L73C61) method instead?